packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss

```diff
@@ -3,6 +3,27 @@
 
 // Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
 
+
+// --------------------------------------------------
+// ChoiceField semantic slots
+
+$radioButton-background-color: $controlBackgroundColor;
```

Comment:
Unfortunately yes. For example, in the selected state, it uses a blue background and a white checkmark, whereas radio buttons have a white background and a blue dot.